src/appmixer/line/core/FindRichMenus/component.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "appmixer.line.core.FindRichMenus",
```

Comment:
this should be `ListRichMenus` instead asit doesn't have any search/ query parameter